aep/general/0111/aep.md

```diff
@@ -0,0 +1,66 @@
+# Planes
+
+Resources and methods on an API can be divided into the _plane_ that they
+reside or perform operations upon. For the context of APIs, the following
+planes are defined:
+
+- Management plane: a uniform, resource-oriented API that primarily configures
+  and allows retrieval of resources.
+- Data plane: a heterogenous API (ideally resource-oriented) that reads and
+  write user data. Often connects to entities provisioned by the management
+  plane, such as virtual machines.
+
+The term "plane" was originally used in networking architecture. Although
+system and network architecture often defines additional planes (e.g. control
+plane or power planes), as the AEPs are focused on the interface, they are not
+defined in this AEP.
+
+## Guidance
+
+### Management Plane
+
+Management resources and methods exist primarily to provision, configure, and
+audit the resources that the data plane interfaces with.
```

Comment:
Does the data plane "interface" with resources?  Or does it contain APIs that contain resources?

Reply:
data plane operates on resources - I'll try to clarify, PTAL.